perun/collect/trace/optimizations/optimization.py

```diff
@@ -3,7 +3,7 @@
 
 import collections
 
-from perun.profile.helpers import sanitize_filepart
+from perun.utils.helpers import sanitize_filepart
```

Comment:
Seems like this import could be merged with the one below it, to keep the imports concise.

Reply:
Good catch!